### PR TITLE
Remove jdk20 api doc and add jdk21

### PR DESCRIPTION
**Pull request by pshipton**
The jdk21 doc is created from the head stream and will be updated when it's time for the initial jdk21 release.

https://openj9-jenkins.osuosl.org/job/Build_JDK21_x86-64_linux_Personal/26/

### CODE REVIEW

**Comment by pshipton** [comment]
jenkins doc stage

**Comment by keithc-ca** [comment]
It seems premature to me to be removing API doc for jdk20 before its replacement (jdk21) is available.

**Comment by pshipton** [comment]
If merged this will go into the draft docs, not the live docs. The draft docs will only go live at the time of the 0.41 release, at which time jdk20 is out of support.

File: docs/api-jdk21.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+-->
+
+# OpenJ9 JDK 20 API documentation
```
Comment (keithc-ca) [inline docs/api-jdk21.md:24]: This should say "21" instead of "20".

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: docs/api-jdk21.md (hunk 1 above)
Comment (pshipton) [reply docs/api-jdk21.md]: Updated.

**Review by keithc-ca — APPROVED** [review]

**Pull request merged**